vital/types/metadata_traits.h

```diff
@@ -135,6 +189,8 @@ class VITAL_EXPORT metadata_traits
   typedef std::shared_ptr< vital_meta_trait_base > trait_ptr;
 #endif
   std::map< kwiver::vital::vital_metadata_tag, trait_ptr> m_trait_table;
+  std::map< std::string, trait_ptr> m_name_trait_table;
+  std::map< std::string, trait_ptr> m_enum_name_trait_table;
```

Comment:
```suggestion
  std::map< std::string, trait_ptr > m_name_trait_table;
  std::map< std::string, trait_ptr > m_enum_name_trait_table;
```

OT, but I wonder if it's safe to assume yet that `std::map` *can* hold a `std::unique_ptr`?